eng/packages/http-client-csharp-mgmt/generator/TestProjects/Local/Mgmt-TypeSpec/bar.tsp

```diff
@@ -52,6 +52,7 @@ model BarSettings is ProxyResource<BarSettingsProperties> {
 
 model BarSettingsProperties {
   /** enabled */
+  @visibility(Lifecycle.Read)
```

Comment:
I don't see an easy way to set up UT to mimic the readonly inner property case, that's why I updated the generation test project instead.